### ISSUE TITLE
Istio 0.2.10 - services showing as unknown

### ISSUE THREAD

**Issue by ekarlso** [istio/istio#1784]
Hi, I am not getting any services in my service group or prometheus. They are all showing unknown. Any ideas on this?

```
// 20171119194359
// http://localhost:8088/graph

{
  "nodes": {
    "bots.default (unknown)": {
      
    },
    "bots.evry-support (unknown)": {
      
    },
    "traefik-ingress-lb.kube-system (unknown)": {
      
    }
  },
  "edges": [
    {
      "source": "bots.default (unknown)",
      "target": "bots.default (unknown)",
      "labels": {
        "reqs/sec": "0.010169"
      }
    },
    {
      "source": "traefik-ingress-lb.kube-system (unknown)",
      "target": "bots.default (unknown)",
      "labels": {
        "reqs/sec": "0.020339"
      }
    },
    {
      "source": "bots.evry-support (unknown)",
      "target": "bots.default (unknown)",
      "labels": {
        "reqs/sec": "0.000000"
      }
    }
  ]
}```

**Comment by sdake** [comment]
@ekarlso Would you mind trying master?  if not master, 0.4.0?  0.2.10 is super old and a whole lotta issues have been resolved since 0.2.10 was tagged.

**Comment by GregHanson** [comment]
resolved by upgrading istio version
